src/containerapp/azext_containerapp/_params.py

```diff
@@ -0,0 +1,186 @@
+        c.argument('registry_password', help='The password of the registry. If using Azure Container Registry, we will try to infer the credentials if not supplied')
+        c.argument('docker_file_path', help='The dockerfile location, e.g. ./Dockerfile')
+        c.argument('service_principal_client_id', help='The service principal client ID. ')
+        c.argument('service_principal_client_secret', help='The service principal client secret.')
+        c.argument('service_principal_tenant_id', help='The service principal tenant ID.')
+
+    with self.argument_context('containerapp github-action delete') as c:
+        c.argument('token', help='A Personal Access Token with write access to the specified repository. For more information: https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line')
+        c.argument('login_with_github', help='Interactively log in with Github to retrieve the Personal Access Token')
+
+    with self.argument_context('containerapp revision') as c:
+        c.argument('revision_name', options_list=['--revision'], type=str, help='Name of the revision.')
+
+    with self.argument_context('containerapp revision copy') as c:
+        c.argument('from_revision', type=str, help='Revision to copy from. Default: latest revision.')
+
+    with self.argument_context('containerapp ingress') as c:
+        c.argument('allow_insecure', help='Allow insecure connections for ingress traffic.')
+        c.argument('type', validator=validate_ingress, arg_type=get_enum_type(['internal', 'external']), help="The ingress type.")
+        c.argument('transport', arg_type=get_enum_type(['auto', 'http', 'http2']), help="The transport protocol used for ingress traffic.")
+        c.argument('target_port', type=int, validator=validate_target_port, help="The application port used for ingress traffic.")
+
+    with self.argument_context('containerapp ingress traffic') as c:
+        c.argument('traffic_weights', nargs='*', options_list=['--traffic-weight'], help="A list of revision weight(s) for the container app. Space-separated values in 'revision_name=weight' format. For latest revision, use 'latest=weight'")
+
+    with self.argument_context('containerapp secret set') as c:
+        c.argument('secrets', nargs='+', options_list=['--secrets', '-s'], help="A list of secret(s) for the container app. Space-separated values in 'key=value' format.")
+
+    with self.argument_context('containerapp secret show') as c:
+        c.argument('secret_name', help="The name of the secret to show.")
+
+    with self.argument_context('containerapp secret remove') as c:
+        c.argument('secret_names', nargs='+', help="A list of secret(s) for the container app. Space-separated secret values names.")
+
+    with self.argument_context('containerapp env dapr-component') as c:
+        c.argument('dapr_app_id', help="The dapr app id.")
```

Comment:
Is this (and other dapr_app_* items) needed here?

```suggestion
        c.argument('dapr_app_id', help="The Dapr app ID.")
```